Test/ModalWindow.test.ts

```diff
@@ -0,0 +1,25 @@
+import modalWindow from '../Src/UIComponents/ModalWindow/ModalWindow';
+
+describe('Modal Window test', () => {
+    test('return HTMLElement', () => {
+        const component = document.createElement('p');
+        const closeWindow = () => console.log('closeWindow');
+        expect(modalWindow(component, closeWindow)).toBeInstanceOf(HTMLElement);
+    })
+
+    test('when background clicked, run function closeWindow()', () => {
+        const component = document.createElement('p');
+        const closeWin = jest.fn();
```

Comment:
I suggest creating closeWin outside of a test function to avoid redeclaring it twice

Reply:
I need to create this function twice because I can not share this between tests (I tried and they failed).